node/src/actors/chain_manager/mod.rs

```diff
@@ -1023,6 +1027,30 @@ impl ChainManager {
 
                 let miner_pkh = block.block_header.proof.proof.pkh();
 
+                // Check total amount staked to make sure we can activate wit/2
+                let superblock_period = chain_info.consensus_constants.superblock_period;
+                if get_protocol_version(Some(block_epoch)) == ProtocolVersion::V1_8
+                    && get_protocol_version_activation_epoch(ProtocolVersion::V2_0) == Epoch::MAX
+                {
+                    if block_epoch % u32::from(superblock_period) == 0 {
+                        if stakes.total_staked() >= Wit::from(POS_MIN_TOTAL_STAKE_NANOWITS) {
+                            register_protocol_version(
+                                ProtocolVersion::V2_0,
+                                block_epoch + 26880,
```

Comment:
We've always hardcoded this amount of epochs (two weeks) in TAPI etc, but yes, it would be cleaner to set it in defaults.rs instead. Note that I'm planning to include all these constants in a proper `consensus_constants_wit2` struct anyway, I just have not gotten around to it yet.